### PR TITLE
feat: Issue templates

### PR DESCRIPTION
**Pull request by PassiHD2004**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Linux]
```
Comment (RoootTheFox) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: most people don't use Linux, add macOS and Windows as example here

**Review by RoootTheFox — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (PassiHD2004) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fair enough

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Desktop (please complete the following information):**
- - OS: [e.g. Linux]
+ - OS: [e.g. Windows 10 22H2]
```
Comment (RoootTheFox) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: gay sex

**Review by RoootTheFox — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (PassiHD2004) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: LMFAO

**Pull request merged**